index.html

```diff
@@ -23,12 +23,12 @@
         <h3>Welcome</h3>
       </a>
     </section>
-    <nav class="menu">
-      <i class="fa fa-bars menu-icon" aria-hidden="true"></i>
-      <ul class="menu-list">
-        <li><a href="#">Portfolio</a></li>
-        <li><a href="#">About</a></li>
-        <li><a href="#">Contact</a></li>
+    <nav class="menu" id="nav">
+      <i id="ham_x" class="fa fa-bars menu-icon" aria-hidden="true"></i>
+      <ul class="menu-list" id="menu-list">
+        <li><a href="#works" onclick="show()">Portfolio</a></li>
+        <li><a href="#about-me" onclick="show()">About</a></li>
+        <li><a href="#contact" onclick="show()">Contact</a></li>
```

Comment:
- [ ] Good job adding the functions that enable the drop-down menu to work!
However, there are a few changes required for you to make here: 
- The nav items, social media icons and links on the page are all missing their required styles. Kindly add them back so they match with the Figma design template.
- The drop-down menu should cover the entire page.
- You are missing a background image decoration in the headline. Kindly add it.
_See image below for reference_

**Original** | **Your clone**
  ----------- | -----------
 ![image](https://user-images.githubusercontent.com/49621279/122804655-044f8f80-d27d-11eb-9359-8a4c5df99e7d.png) | ![image](https://user-images.githubusercontent.com/49621279/122804705-13364200-d27d-11eb-8713-8a127b9cd6e1.png) 
![image](https://user-images.githubusercontent.com/49621279/122804977-5f818200-d27d-11eb-99a6-50b3ac09fa47.png) | ![image](https://user-images.githubusercontent.com/49621279/122805019-6c05da80-d27d-11eb-8603-569d12f3b44b.png)
![image](https://user-images.githubusercontent.com/49621279/122804448-ca7e8900-d27c-11eb-9e6b-9b8c3d4c0a21.png) | ![image](https://user-images.githubusercontent.com/49621279/122804539-e4b86700-d27c-11eb-9778-c62800c661ba.png)
![image](https://user-images.githubusercontent.com/49621279/122805213-aa9b9500-d27d-11eb-925d-2cd1a8e9cefa.png) | ![image](https://user-images.githubusercontent.com/49621279/122805415-ec2c4000-d27d-11eb-87e5-9e28113a7412.png)

Reply:
* As the previous reviewer pointed out your design social media icon links still look different 
Yours:
![Screenshot (92)](https://user-images.githubusercontent.com/52497246/122809918-a2b41300-d2c6-11eb-8260-782fc64d3924.png)
____________________________________________________________________________________________________________________________

![Screenshot (90)](https://user-images.githubusercontent.com/52497246/122810091-d8f19280-d2c6-11eb-9f54-c50c90647eb5.png)
________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________
Figma:
![Screenshot (91)](https://user-images.githubusercontent.com/52497246/122810220-00485f80-d2c7-11eb-8995-6a05852fa70c.png)

![Screenshot (89)](https://user-images.githubusercontent.com/52497246/122810249-09d1c780-d2c7-11eb-8aad-7a2424ac2d72.png)